anda/devs/ghostty/nightly/update.rhai

```diff
@@ -1,5 +1,8 @@
+let html = get(`https://raw.githubusercontent.com/ghostty-org/ghostty/refs/heads/main/build.zig.zon`);
+let ver = find(".version = \"([\\d.]+)\"", html, 1);
```

Comment:
when `rpm.changed()` is `false`, this is not needed, so you are doing something useless most of the time

you should put this inside the if branch